pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -273,9 +273,9 @@ func (r *Reconciler) fetchAllResourcesInOneNamespace(namespaceName string, place
 		if !r.shouldSelectResource(gvr) {
 			continue
 		}
-		if !r.InformerManager.IsInformerSynced(gvr) {
-			return nil, controller.NewExpectedBehaviorError(fmt.Errorf("informer cache for %+v is not synced yet", gvr))
-		}
+		//if !r.InformerManager.IsInformerSynced(gvr) {
+		//	return nil, controller.NewExpectedBehaviorError(fmt.Errorf("informer cache for %+v is not synced yet", gvr))
+		//}
```

Comment:
Had to change to use GVK to be able to get the informer but GVK does not used in this function